e2e/pages/events/manageDocumentsEvent.page.js

```diff
@@ -65,10 +65,42 @@ module.exports = {
     I.attachFile(this.fields.supportingDocuments(index).document, document);
   },
 
+  async selectFurtherEvidenceType(type, index = 0) {
+    switch (type) {
+      case 'Expert reports':
+        I.checkOption(this.fields.supportingDocuments(index).type.expert);
+        break;
+      case 'Other reports':
+        I.checkOption(this.fields.supportingDocuments(index).type.other);
+        break;
+      default:
+        throw new Error(`Unsupported further evidence type ${type}`);
+    }
+  },
+
   async selectConfidential(index = 0) {
     I.click(this.fields.supportingDocuments(index).confidential);
   },
 
+  async uploadFurtherEvidence(supportingEvidenceDocument) {
```

Comment:
could be done better, basically same as uploadSupportingEvidenceDocument with one line different

Reply:
yeah, can be single method with additional parameter with default value

async uploadFurtherEvidence(supportingEvidenceDocument, confidential=false) {
....
if(confidential){
this.selectConfidential(index);
}
}

or 

async uploadConfidentialFurtherEvidence(supportingEvidenceDocument) {
  await uploadFurtherEvidence(supportingEvidenceDocument) ;
  this.selectConfidential(index);
  
}

Reply:
now two methods, one for normal one for confidential (with parameter to specify if further evidence type should be set)